tests/tests.js

```diff
@@ -1,14 +1,79 @@
 const tape = require('tape');
-const request = require('supertest');
+const supertest = require('supertest');
 const server = require('../server.js');
+const axios = require('axios');
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+
+const postPlace = require('../routes/middleware/post_place.js');
 
 tape('home route test: GET request to /', t => {
-  request(server)
+  supertest(server)
     .get('/')
     .end((err, res) => {
-      if (err) throw err;
+      t.error(err);
       t.equal(res.status, 200, 'should return status code 200');
-      t.ok(res.text.includes('<div id="list-page-content">'), 'should return home page containing test string "<div id="list-page-content">"');
+      t.ok(res.text.includes('<div id="list-page-content">'));
+      t.end();
+    });
+});
+
+tape('Test the event details route', async t => {
+  let eventId = '';
+  try {
+    const eventResponse = await axios.get(
+      'https://nazareth-open-tourism-platform.herokuapp.com/api/v1/events'
```

Comment:
You should probably have two environments, a dev setup that's running a local version of the OTP and a production one that's linked to the real OTP. That would fix two important problems:

1. Not being able to mess around with the production database (add dummy events within tests, etc.)
1. Having to wait x seconds for your tests to run.